test/tools/unified-spec-runner/runner.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
I made it global here because I wanted to take a consistent approach to ts inside the unified runner, and there are way too many instances of it in match to inline; if we decide that the non-null assertions are an acceptable approach inside our test runner, we can disable the rule explicitly in the ts config for testing and then it'll be easier to delete if it's only declared in one place. If we decide that the non-null assertions are not acceptable in general in testing but ok sometimes, we can make the change to inline it then. But since we haven't decided on a convention, I went with the approach that's easiest to revert.